# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/nsCryptoHash.h
# Commit: 63b555e1f503
# Full Hash: 63b555e1f503db91a54604fd8cef88dfab7c0c7a
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2023-06-08 09:27:30
# Regressor Bug: 1834862
# File Overlap Count: 1
# Description:
#   Bug 1834862 - implement nsICryptoHash in rust r=jschanck,supply-chain-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D179727
# ==============================================================================

diff -r 6337ce701771 -r 63b555e1f503 security/manager/ssl/nsCryptoHash.h
--- a/security/manager/ssl/nsCryptoHash.h	Wed Jun 07 22:44:15 2023 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,38 +0,0 @@
-/* -*- Mode: C++; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*-
- *
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#ifndef nsCryptoHash_h
-#define nsCryptoHash_h
-
-#include "ScopedNSSTypes.h"
-#include "hasht.h"
-#include "nsICryptoHash.h"
-#include "secmodt.h"
-
-class nsIInputStream;
-
-#define NS_CRYPTO_HASH_CID                           \
-  {                                                  \
-    0x36a1d3b3, 0xd886, 0x4317, {                    \
-      0x96, 0xff, 0x87, 0xb0, 0x00, 0x5c, 0xfe, 0xf7 \
-    }                                                \
-  }
-
-class nsCryptoHash final : public nsICryptoHash {
- public:
-  NS_DECL_ISUPPORTS
-  NS_DECL_NSICRYPTOHASH
-
-  nsCryptoHash();
-
- private:
-  virtual ~nsCryptoHash() = default;
-
-  mozilla::UniqueHASHContext mHashContext;
-  bool mInitialized;
-};
-
-#endif  // nsCryptoHash_h